src/lib/storage/chunk.cpp

```diff
@@ -102,6 +102,17 @@ std::vector<std::shared_ptr<AbstractIndex>> Chunk::get_indexes(
   return result;
 }
 
+void Chunk::update_max_begin_cid() {
+  DebugAssert((has_mvcc_data()), "Chunk does not have mvcc data");
+  auto mvcc = get_scoped_mvcc_data_lock();
+
+  if (mvcc->begin_cids.empty()) {
+    return;
```

Comment:
When would has_mvcc_data be true and begin_cids be empty?

Reply:
I don't know. Could it be that the chunk points to MVCC data but does not have entries yet?

Reply:
No, I'd assert for it to not be empty.